packages/ui/src/working-groups/components/SelectWorkingGroup/OptionsListWorkingGroup.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
 export const OptionsListWorkingGroup = React.memo(({ allWorkingGroups, onChange }: Props) => (
   <OptionsListComponent>
     {allWorkingGroups.map((option) => (
-      <Option key={option.id} onClick={() => onChange(option)}>
+      <Option key={option.id} onClick={() => onChange(option)} disabled={!option.leadId}>
```

Comment:
Too general ;)

You should be able to select Working Group for "Create Working Group Lead Opening" Proposal

It should be disabled for:

*   Decrease Working Group Lead Stake
*   Slash Working Group Lead